sklearn/gaussian_process/kernels.py

```diff
@@ -572,8 +572,8 @@ def __call__(self, X, Y=None, eval_gradient=False):
             is evaluated instead.
 
         eval_gradient : bool, default=False
-            Determines whether the gradient with respect to the kernel
-            hyperparameter is determined.
+            Determines whether the gradient with respect to the log of the
+            kernel hyperparameter is determined.
```

Comment:
can we change "is determined" to "is computed" here and below, while we are at it?

Reply:
@jnothman  sure thing. Just pushed that.